apps/demo-ui/ui/src/OpenSearch.tsx

```diff
@@ -126,8 +126,12 @@ export const getEmbeddingModels = async () => {
                         "field": "chunk_number"
                     }
                 },
-                "must": [
+                "minimum_should_match": 1,
+                "should": [
                     { "term": { "algorithm": "TEXT_EMBEDDING" } },
+                    { "match_phrase": { "description": "embedding" } },
```

Comment:
At some point we should just display the model name in the dropdown instead of only the id